src/uct/ib/rc/base/rc_ep.h

```diff
@@ -185,7 +193,7 @@ static inline void uct_rc_ep_process_tx_completion(uct_rc_iface_t *iface,
 
     ucs_queue_for_each_extract(op, &ep->outstanding, queue,
                                UCS_CIRCULAR_COMPARE16(op->sn, <=, sn)) {
-        op->handler(op);
+        op->handler(op, UCS_OK);
```

Comment:
but shouldn't we call use completion with error status? for example get_bcopy, atomics, zero copy operations...